### DISCUSSION TITLE
matrix with bignumbers  .ToString()

### DISCUSSION THREAD

**Discussion by nycos62** [josdejong/mathjs#3331] [Q&A] [answered]
Hello, 
I'm probably missing something while using toString()
is there a way to force toString() method to output toFixed() for bignumbers and not the scientific display

generating code :
math.kron(math.range(math.bignumber("11111"),math.bignumber("11211")),math.range(math.bignumber("11111"),math.bignumber("11211"))).toString()

output 1.24387809e+8 instead of 124387809

thank you

**Comment by josdejong** [comment, marked as answer]
You can use the function `format` intsead of `toString`:

```js
const result = math.kron(
  math.range(math.bignumber("11111"), math.bignumber("11211")),
  math.range(math.bignumber("11111"), math.bignumber("11211"))
)

console.log(math.format(result, { upperExp: Infinity }))
// [[123454321, 123465432, 123476543, 123487654, 123498765, ... 
``` 

Docs: https://mathjs.org/docs/reference/functions/format.html
